site/layouts/partials/head.html

```diff
@@ -49,49 +49,11 @@
 <link rel="stylesheet" href="{{ "css/note.css" | absURL }}">
 
 <script async defer src="https://buttons.github.io/buttons.js"></script>
-<script async src="//cdn.bizible.com/scripts/bizible.js"></script>
```

Comment:
Same here

Reply:
Same applies. Let me know if you have any questions. I can probably get you access to the GTM account if you'd like to review it there.

Reply:
Got it. As long as they are in GTM, I'M down.